switch.py

```diff
@@ -34,17 +34,95 @@ async def async_setup_entry(
         ]
     if router.tailscale_configured:
         switches.append(TailscaleSwitch(router))
+    for iface in router._wifi_ifaces:
+        switches.append(WifiApSwitch(router, iface))
```

Comment:
the `iface` names are ugly and do not reflect the GLinet UI
![image](https://github.com/user-attachments/assets/f2752e90-b483-4a70-b1d7-98df2cdb08d4)

But ssid may not be unique
```suggestion
    for key, iface in router._wifi_ifaces.items():
        switches.append(WifiApSwitch(router, key, iface))
```
so see changes below